DataFormats/PatCandidates/interface/Electron.h

```diff
@@ -32,6 +32,8 @@
 #include "DataFormats/PatCandidates/interface/PackedCandidate.h"
 #include "DataFormats/Common/interface/AtomicPtrCache.h"
 
+#include "PhysicsTools/PatUtils/interface/MiniIsolation.h"
```

Comment:
MiniIsolation definition has to be moved to DataFormats/ for the data member parts of it.
The part that implements the algorithm reading from a pf collection will stay in PhysicsTools